views/backend/dashboard.php

```diff
@@ -50,52 +19,44 @@
         <?php   
         if(!empty($comments)) {
             foreach ($comments as $comment) {
-
-                if(filter_has_var(INPUT_GET, 'delete')){
-                    $comment = $model_comment->delete_comment();
-                }
-                
-                if(filter_has_var(INPUT_GET, 'approve')){
-                    $comment = $model_comment->approve_comment();
-                }
         ?>
         
         <div>
-                <tr id="commentaire_<?=$comment->id?>">
-                    <td><?= $comment->title ?></td>
-                    <td><?= substr($comment->comment, 0, 100); ?>...</td>
+                <tr id="commentaire_<?=$comment->getId?>">
+                    <td><?= $comment->getName() ?></td>
+                    <td><?= substr($comment->getComment(), 0, 100); ?>...</td>
                     <td>
 
-                        <a href="index.php?page=dashboard&approve=<?= $comment->id ?>" type="submit" 
+                        <a href="index.php?page=dashboard&approve=<?= $comment->getId() ?>" type="submit" 
                             class="btn-floating btn-small waves-effect waves-light green see_comment" 
                             onclick="return window.confirm(`Êtes vous sur de vouloir d'approuver ce commentaire ?!`)"><i
                             class="material-icons">done</i></a>
 
-                        <a href="index.php?page=dashboard&delete=<?= $comment->id ?>" type="submit" 
+                        <a href="index.php?page=dashboard&delete=<?= $comment->getId() ?>" type="submit" 
                             class="btn-floating btn-small waves-effect waves-light red see_comment" 
                             onclick="return window.confirm(`Êtes vous sur de vouloir supprimer ce commentaire ?!`)"><i
                             class="material-icons">delete</i></a>
         
-                        <a href="#comment_<?= $comment->id ?>"
+                        <a href="#comment_<?= $comment->getId() ?>"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$comment'.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)